packages/server/validators/auth.js

```diff
@@ -0,0 +1,27 @@
+const { body } = require("express-validator");
+
+const forgotPasswordValidator = [
+  body("email", "Invalid email.")
+    .trim()
+    .exists({ checkFalsy: true })
+    .withMessage("Email can't be null or empty.")
+    .isEmail()
+    .withMessage("Invalid email entered!"),
+];
+
+const passwordResetValidator = [
+  body("token", "Invalid token.")
+    .exists({ checkFalsy: true })
+    .withMessage("Token can't be null or empty.")
+    .trim(),
+
+  body("password", "Invalid password.")
+    .exists({ checkFalsy: true })
+    .withMessage("Password can't be null or empty.")
+    .trim(),
```

Comment:
The order of your validator checks matter. Trim first and then check truthiness. The way you have it right now, a string with just whitespaces `"        "` will pass because it's a truthy value and then you're trimming it after checking truthiness.